CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 - The DNS configuration can now be passed to the pod via the command line ([#2040](https://github.com/coreos/rkt/pull/2040)). See [`DNS support`](https://github.com/coreos/rkt/blob/master/Documentation/networking.md#dns-support) documentation.
 - Errors are now structured, allowing for better control of the output. See [#1937](https://github.com/coreos/rkt/pull/1937) for background and the [Error & Output](https://github.com/coreos/rkt/blob/master/Documentation/hacking.md#errors--output) section of the Hacking document for usage.
 - All output now uses the new log package in `pkg/log` to provide a more clean and consistent output format and more helpful debug output. See [#1937](https://github.com/coreos/rkt/pull/1937)
+- Added configuration for stage1 image. User can drop a configuration file to `/etc/rkt/stage1.d` (or to `stage1.d` in user configuration directory) to tell rkt to use different name, version and location of stage1 image instead of build-time defaults ([#1977](https://github.com/coreos/rkt/pull/1977)).
```

Comment:
User**s** can...

"in the user configuration directory"

"to tell rkt to use a different stage1 image name, version and location instead of..."